allwinner/h5.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2022 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// This file contains pin mapping information that is specific to the Allwinner
+// H5 model.
+
+package allwinner
+
+import (
+	"strings"
+
+	"periph.io/x/conn/v3/pin"
+	"periph.io/x/host/v3/sysfs"
+)
+
+// mappingH5 describes the mapping of the H5 processor gpios to their
+// alternate functions.
+//
+// It omits the in & out functions which are available on all gpio.
+//
+// The mapping comes from the datasheet page 23:
```

Comment:
Page 23?

Reply:
Good catch, I forgot to update the page number.

Reply:
Updated in latest commit